### DISCUSSION TITLE
[python flask] Is it normal use about 10% of single CPU core using skywalking without request?

### DISCUSSION THREAD

**Discussion by smartmark-pro** [apache/skywalking#9426] [Q&A] [answered]
>top -c
![image](https://user-images.githubusercontent.com/28584805/182820303-1881eee2-3338-4dc6-9f51-5de29d12e213.png)
>top -H -p 14691
![image](https://user-images.githubusercontent.com/28584805/182820413-b24e5d28-754e-42b2-a6be-d6e3113fd5f4.png)
>py-spy top --pid 14691
![image](https://user-images.githubusercontent.com/28584805/182820567-74decac5-342d-4b16-82eb-c91dfa296d0d.png)
>strace -p 14691
![image](https://user-images.githubusercontent.com/28584805/182821665-e1872073-d9a2-471a-bc32-7ca5ef435c5d.png)
>gdb python 
>run app.py
![image](https://user-images.githubusercontent.com/28584805/182824728-24afab08-49d7-471c-8bca-0aeb94efbf09.png)

If this is not normal, I would spend time to confirm the problem

**Comment by smartmark-pro** [comment, marked as answer]
python3.6 下指定grpcio==1.31.0不会出现该问题, 默认会装最新的依赖包,

**Reply by Superskyyy** [reply]
> python3.6 下指定grpcio==1.31.0不会出现该问题, 默认会装最新的依赖包,

Hello, I think it's something on polling from grpc side that caused the high cpu usage, so you are right either downgrade or bump up to after v1.51.0
